modules/lb-controller/main.tf

```diff
@@ -35,11 +35,11 @@ resource "helm_release" "lbc" {
   cleanup_on_fail = lookup(var.helm, "cleanup_on_fail", true)
 
   dynamic "set" {
-    for_each = {
+    for_each = merge({
       "clusterName"                                               = var.cluster_name
       "serviceAccount.name"                                       = local.serviceaccount
       "serviceAccount.annotations.eks\\.amazonaws\\.com/role-arn" = module.irsa[0].arn[0]
-    }
+    }, lookup(var.helm, "vars"))
```

Comment:
This code change allows you to pass an additional helm variable to support the lb-controller for the fargate profile. This is the related pull request:  https://github.com/Young-ook/terraform-aws-eks/pull/138.

@patradinesh, if you have chance to review, please provide your feedback. thank you.